libkbfs/rekey_fsm_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"context"
+
+	"github.com/keybase/kbfs/tlf"
+)
+
+func requestRekeyWithContextAndWaitForOneFinishEvent(
```

Comment:
Seems like you wouldn't need this function (or file) if you follow my suggestion above.